armi/reactor/tests/test_components.py

```diff
@@ -508,6 +513,50 @@ def test_getAreaColdTrue(self):
         self.assertAlmostEqual(totalAreaCold, totalAreaHot, delta=1e-10)
 
 
+class TestComponentList(unittest.TestCase):
+    def setUp(self):
+        self.componentList = []
```

Comment:
Generally not advisable to embed variable types in their names. See #1881 

This `TestCase` is responsible for testing the sorting of various components
```suggestion
class TestComponentSort(unittest.TestCase):
    def setUp(self):
        self.components = []
```